parlai/core/agents.py

```diff
@@ -519,7 +499,7 @@ def compare_init_model_opts(opt: Opt, curr_opt: Opt):
         print('*' * 75)
 
 
-def load_agent_module(opt: Opt):
+def create_agent_from_file_opt(opt: Opt):
```

Comment:
super nit: for some reason i feel like `create_agent_from_opt_file` sounds better here, unless there's something i'm missing

Reply:
yes it definitely does lol...idk what i was thinking